File: GTTS/INL.VettingService.Models/PersonVettingVettingType_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class PersonVettingVettingType_Item : IPersonVettingVettingType_Item
    {
        public string FirstMiddleNames { get; set; }
        public string LastNames { get; set; }
        public DateTime? DOB { get; set; }
        public long PersonsVettingID { get; set; }
        public int VettingTypeID { get; set; }
        public string VettingTypeCode { get; set; }
        public bool CourtesyVettingSkipped { get; set; }
        public string CourtesyVettingSkippedComments { get; set; }
        public byte? HitResultID { get; set; }
        public string HitResultDetails { get; set; }
        public string HitResultCode { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/person-vetting-vetting-type_item.ts


import { IPersonVettingVettingType_Item } from './iperson-vetting-vetting-type_item';

export class PersonVettingVettingType_Item implements IPersonVettingVettingType_Item {
  
	public FirstMiddleNames: string = "";
	public LastNames: string = "";
	public DOB?: Date;
	public PersonsVettingID: number = 0;
	public VettingTypeID: number = 0;
	public VettingTypeCode: string = "";
	public CourtesyVettingSkipped: boolean = false;
	public CourtesyVettingSkippedComments: string = "";
	public HitResultID?: number;
	public HitResultDetails: string = "";
	public HitResultCode: string = "";
  
}


